.github/workflows/ci.yml

```diff
@@ -25,18 +25,18 @@ jobs:
   matrix:
     uses: ./.github/workflows/run-matrix.yml
     with:
-      versionsFile: .ci/.jenkins_ruby.yml
-      frameworksFile: .ci/.jenkins_framework.yml
-      excludedFile: .ci/.jenkins_exclude.yml
+      versionsFile: .ci/.ruby.yml
```

Comment:
no more jenkins references, a bit unrelated to the nature of this change, but since it's the very last thing before removing jenkins in this project, then let's include it